apis/core-api/src/api/routes/v1/awsS3/ControllerAdapter.ts

```diff
@@ -0,0 +1,23 @@
+import AwsS3Controller from 'core/controllers/AwsS3Controller';
```

Comment:
Please create `aws` folder inside controllers, and create `S3Controller` inside, so you will need to change routes to /aws/s3 as well, you may create `aws` folder inside routes. This is because there may be different aws services controllers, and need to separate them for readibility

Reply:
okay, will do